packages/utils/src/uri.ts

```diff
@@ -28,12 +28,14 @@ export function parseUri(str: string): EngineTypes.UriParameters {
   const requiredValues = path.split("@");
   const queryString: string = typeof pathEnd !== "undefined" ? str.substring(pathEnd) : "";
   const queryParams = qs.parse(queryString);
+  const methods = queryParams.methods ? (queryParams.methods as string).split(",") : undefined;
```

Comment:
Consider using a constant for both splitting and joining.
You can also avoid casting the methods to a string if you change your initial condition like below.

```suggestion
  const METHODS_SEPARATOR = ","
  const methods = typeof queryParams.methods === 'string' ? queryParams.methods.split(METHODS_SEPARATOR) : undefined;
```